src/conf_mode/service_dhcp-server.py

```diff
@@ -286,6 +287,15 @@ def verify(dhcp):
                             f'DHCP static-route "{route}" requires router to be defined!'
                         )
 
+            # If a client class has been specified then it must exist
+            if 'client_class' in subnet_config:
+                client_class = subnet_config['client_class']
+                if 'client_class' not in dhcp:
+                    raise ConfigError(f'Client class "{client_class}" set in subnet "{subnet}" but does not exist')
+
+                if client_class not in dhcp['client_class'].keys():
+                    raise ConfigError(f'Client class "{client_class}" set in subnet "{subnet}" but does not exist')
```

Comment:
```suggestion
                if client_class not in dhcp.get('client_class', {}):
                    raise ConfigError(f'Client class "{client_class}" set in subnet "{subnet}" but does not exist')
```

This part can by simplified